GPT/gpt-settings.talon

```diff
@@ -8,4 +8,4 @@ settings():
     user.openai_model = 'gpt-3.5-turbo'
 
 # Only uncomment the line below if you want experimental behavior to parse Talon files
-tag(): user.gpt_beta
+# tag(): user.gpt_beta
```

Comment:
Let's resolve this merge conflict so we use the `.example` config pattern instead of this.

Reply:
done!